packages/mobile/src/send/SendAmount.tsx

```diff
@@ -253,6 +277,13 @@ export class SendAmount extends React.Component<Props, State> {
       return
     }
 
+    const now = Date.now()
```

Comment:
Why pull now here and then pass it in instead of getting it in `isPaymentLimitReached`?

Reply:
I think this would make unit testing nicer, instead of having to deal with time traveling libraries? it is kind of weird but then the function itself is more deterministic was my thinking. wdyt?

Reply:
another benefit is that we need the time when getting the amountRemaining and by passing it into both we are certain to use the same time in both place otherwise we could technically have people who try and send 24.5999 hours after sending enough to put them over get an error but the the error message will say they have some remaining.